apps-rendering/src/components/layout/live.tsx

```diff
@@ -102,42 +104,60 @@ interface Props {
 	item: LiveBlog | DeadBlog;
 }
 
-const Live: FC<Props> = ({ item }) => (
-	<article className="js-article">
-		<LiveblogHeader item={item} />
-		<main css={mainStyles}>
-			<GridItem area="metadata">
-				<div css={metadataWrapperStyles(item)}>
-					<Metadata item={item} />
-				</div>
-			</GridItem>
-			<GridItem area="key-events">
-				<div css={keyEventsWrapperStyles}>
-					<KeyEvents
-						keyEvents={keyEvents(item.blocks)}
+const Live: FC<Props> = ({ item }) => {
+	const pagination = (
+		<Pagination
```

Comment:
Looks great! Hope the integration with the shared `Pagination` component wasn't too tricky?

Reply:
It lined up perfectly 😄 - you and @marjisound did a brilliant job!